README.md

```diff
@@ -20,6 +20,41 @@ and review the napari docs for plugin developers:
 https://napari.org/docs/plugins/index.html
 -->
 
+## Local variables
+
+In napari-console 0.0.8 and earlier, the console `locals()` namespace only
+contained a reference to the napari viewer that enclosed the console.
+
+Since version 0.0.9, it instead contains everything in the enclosing frame that
+called napari. That is, if your Python code is:
+
+```python
+import napari
+import numpy as np
+from scipy import ndimage as ndi
+
+image = np.random.random((500, 500))
+labels = ndi.label(image > 0.7)[0]
+
+viewer, image_layer = napari.imshow(image)
+labels_layer = viewer.add_labels(labels)
+
+napari.run()
+```
+
+Then the napari console will have the variables `np`, `napari`, `ndi`, `image`,
+`labels`, `viewer`, `image_layer`, and `labels_layer` in its namespace.
+
+This is implemented by inspecting the Python stack when the console is first
+instantiated, finding the first frame that is outside of the `napari_console`,
+`napari`, and `in_n_out` modules, and passing the variables in the frame's
+`f_locals` and `f_globals` to the console namespace.
+
+If you want to disable this behavior (for example, because you are embedding
+napari and the console within some larger application), you can add
+`NAPARI_EMBED=1` to your environment variables before instantiating the
+console.
```

Comment:
This is not true. I implement it as global module-level variable, not an environment variable. Should I change it?

Reply:
Oh! Sorry. I did read that and thought you misspoke. 😅 I think I prefer this way because it's easier to describe and quite a common way to change functionality. Otherwise you have to make sure that the global is accessible in the correct frame, which is not something people are used to thinking about, I think? 🤷

Reply:
yes, but environment variable is global per process, and module level variable is in module. 

If someone would like first spawn PartSeg and then napari, If I use an environment variable, then it will prevent variable propagation in follow-up napari start. 

If using module-level variable it impacts only possible console spawn when open console from napari embedded here. But will not impact napari spawned from the top-level script.